### PR TITLE
Fixing sleep

### PR DESCRIPTION
**Pull request by fulmicoton**
Removing the concept of jump a given amount of time.
Instead, we just have Universe::sleep and a mode in which
the time is accelerated.

### CODE REVIEW

File: quickwit/quickwit-actors/src/universe.rs [hunk 1]
```diff
@@ -81,17 +95,8 @@ impl Universe {
...
-    /// This is smarter than just mocking a clock.
-    /// As long as everything is happening in actors handlers, initialize
-    /// or finalize methods, the result should be rigorously the same as
-    /// if we waited.
-    pub async fn simulate_time_shift(&self, duration: Duration) {
-        self.spawn_ctx
-            .scheduler_client
-            .simulate_sleep(duration)
-            .await;
+    pub async fn sleep(&self, duration: Duration) {
```
Comment (evanxg852000) [inline quickwit/quickwit-actors/src/universe.rs:98]: The `simulate_time_shift` seems a better name for this IMHO. We need the bring back the updated docs comment

**Review by evanxg852000 — APPROVED** [review]
LGTM :+1: 
See my suggestions and looking forward to your take on some of my comments.

File: quickwit/quickwit-aws/src/retry.rs [hunk 2]
```diff
@@ -162,12 +162,13 @@ mod tests {
     #[async_trait]
     impl super::MockableTime for SchedulerClient {
         async fn sleep(&self, duration: Duration) {
-            self.simulate_sleep(duration).await;
+            SchedulerClient::sleep(&self, duration).await;
```
Comment (evanxg852000) [inline quickwit/quickwit-aws/src/retry.rs:165]:
```suggestion
            SchedulerClient::sleep(self, duration).await;
```

File: quickwit/quickwit-actors/src/universe.rs (hunk 1 above)
Comment (fulmicoton) [reply quickwit/quickwit-actors/src/universe.rs:98]: It is not the same semantics as it used to be. it does not accelerate time in itself anymore.

Instead, its semantics is "just sleep".

But the universe can created with an accelerated time.

**Pull request merged**